src/Enhavo/Bundle/NewsletterBundle/Controller/NewsletterController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Enhavo\Bundle\NewsletterBundle\Controller;
+
+use Enhavo\Bundle\AppBundle\Controller\ResourceController;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class SubscriberController extends ResourceController
```

Comment:
Why this class called SubscriberController, but is in the file NewsletterController.php? I think this won't work!